configs/analytical/reactions.json

```diff
@@ -7,14 +7,26 @@
       "reactions":
       [
         {
-          "type": "ARRHENIUS", "A": 0.00012, "B": 7, "C" : 75, "D": 50, "E": 0.5,
-          "reactants": {"B": {"qty": 1}},
-          "products": {"C": {"yield": 1}}
+          "type": "USER_DEFINED",
+          "MUSICA name" : "reaction 2",
+          "reactants": {"E": {"qty": 1}},
+          "products": {"F": {"yield": 1}}
+        },
+        {
+          "type": "USER_DEFINED",
+          "MUSICA name" : "reaction 1",
+          "reactants": {"D": {"qty": 1}},
+          "products": {"E": {"yield": 1}}
```

Comment:
My only concern with this is that `USER_DEFINED` isn't technically part of the camp configuration. However, I do think this is incredibly useful. We could remove photolysis, first order loss, emission, from the technical specification of the configuration because they are all just logical versions of user defined and it allows us to not have to do funny things with music box for integrated reaction rates.

I guess there's nothing actionable with that paragraph; I'm just putting thoughts out there